docs/manual/system/settings.de.md

```diff
@@ -199,12 +199,11 @@ parameters:
 ## config.yml
 
 Die normale Bundle Config gehört in die `config.yml` und befindet sich im Ordner `app/config/`. 
-Falls die Datei noch nicht vorhanden ist, muss diese angelegt werden. Contao lädt automatisch die `config_prod.yml` 
-bzw. `config_dev.yml` und falls nicht vorhanden die `config.yml`.
+Falls die Datei noch nicht vorhanden ist, kann diese angelegt werden. Contao lädt automatisch die `config_prod.yml` 
+bzw. `config_dev.yml` und falls nicht vorhanden die `config.yml` und falls auch die nicht vorhanden die Standard-Konfiguration.
```

Comment:
Die "Standard-Konfiguration" wird sowieso geladen, egal ob du eine `config.yml` anlegst oder nicht. In deiner eigenen `config.yml` kannst du die "Standard-Konfiguration" _ändern_.